manifests/z/Zidoro/Pomatez/1.3.1/Zidoro.Pomatez.locale.en-US.yaml

```
@@ -0,0 +1,33 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Zidoro.Pomatez
+PackageVersion: 1.3.1
+PackageLocale: en-US
+Publisher: Zidoro

```

```suggestion
Publisher: Roldan Montilla Jr
```
Must match Apps&Features / Control Panel